packages/dev/core/src/Materials/Textures/Filtering/hdrFiltering.ts

```diff
@@ -7,10 +7,11 @@ import { Constants } from "../../../Engines/constants";
 import { EffectWrapper, EffectRenderer } from "../../../Materials/effectRenderer";
 import type { Nullable } from "../../../types";
 import type { RenderTargetWrapper } from "../../../Engines/renderTargetWrapper";
+import { Logger } from "../../../Misc/logger";
 
+import "../../../Engines/Extensions/engine.renderTargetCube";

```

Comment:
How does it work in a WebGPU only context? This import will inject some methods in `ThinEngine.prototype`, but if we want to only use WebGPU in a ES6 context, we won't have `ThinEngine` defined?

Reply:
I do admit that the import in WebGpu will be hard to get so I will work on a fix ASAP